ui/css/perf.css

```diff
@@ -49,6 +49,11 @@ to { opacity: 0; }
   color: #999
 }
 
+.graph-legend.seriesinactive {
```

Comment:
@MikeLing perhaps seriesinactive could be hyphenated. eg. `.series-inactive`. Though wait until @wlach gets back tomorrow to confirm.